libraries/AP_Math/control.cpp

```diff
@@ -508,18 +508,45 @@ float input_expo(float input, float expo)
     return input;
 }
 
-/*
-  convert a maximum lean angle in degrees to an accel limit in m/s/s
- */
+// angle_to_accel converts a maximum lean angle in degrees to an accel limit in m/s/s
 float angle_to_accel(float angle_deg)
 {
     return GRAVITY_MSS * tanf(radians(angle_deg));
 }
 
-/*
-  convert a maximum accel in m/s/s to a lean angle in degrees
- */
+// accel_to_angle converts a maximum accel in m/s/s to a lean angle in degrees
 float accel_to_angle(float accel)
 {
     return degrees(atanf((accel/GRAVITY_MSS)));
 }
+
+// rc_input_to_roll_pitch - transform pilot's normalised roll or pitch stick input into a roll and pitch euler angle command
+// roll_in_unit and pitch_in_unit - are normalised roll and pitch stick input
+// angle_max_deg - maximum lean angle from the z axis
+// angle_limit_deg - provides the ability to reduce the maximum output lean angle to less than angle_max_deg
+// returns roll and pitch angle in degrees
+void rc_input_to_roll_pitch(float &roll_out_deg, float &pitch_out_deg, float roll_in_unit, float pitch_in_unit, float angle_max_deg, float angle_limit_deg)
```

Comment:
It's a bit unconventional (at least in AP) to have the outputs as the first arguments. Normally they would be at the end I think.  @peterbarker do you have an opinion on this?

Reply:
I concur - outputs last in ArduPilot (I've had to make the same change in other PRs).

Esp. so in this case as the method name is termed as "to" - having the arguments in the same order as in the method name is something I like.

Reply:
Point well made! I will keep this in mind in the future!